test/e2e/apimachinery/aggregator.go

```diff
@@ -516,7 +518,7 @@ func TestSampleAPIServer(ctx context.Context, f *framework.Framework, aggrclient
 	var jr *apiregistrationv1.APIService
 	err = json.Unmarshal([]byte(statusContent), &jr)
 	framework.ExpectNoError(err, "Failed to process statusContent: %v | err: %v ", string(statusContent), err)
-	framework.ExpectEqual(jr.Status.Conditions[0].Message, "all checks passed", "The Message returned was %v", jr.Status.Conditions[0].Message)
+	gomega.Expect(jr.Status.Conditions[0].Message).To(gomega.Equal("all checks passed"), "The Message returned was %v", jr.Status.Conditions[0].Message)
```

Comment:
There are ways to make statements about a collection (`jr.Status.Conditions` in this case`), but let's not get carried away - this is fine.